src/server/report_exception.cpp

```diff
@@ -48,6 +51,15 @@ void mir::report_exception(std::ostream& out_stream, std::ostream& err_stream)
         }
 
         err_stream << "ERROR: " << boost::diagnostic_information(error) << std::endl;
+
+        // Log a security event
+        std::string function_str = "<function unknown>";
+        if (auto function = boost::get_error_info<boost::throw_function>(error))
+        {
+            function_str = *function;
+        }
+        auto message = std::format("Mir unhandled exception in: {}", function_str);
+        err_stream << mir::security_fmt(mir::logging::Severity::error, "sys_crash", message) << std::endl;
```

Comment:
There is no reason to report this exception is unhandled.

E.g. in `miral::MirRunner::Self::run_with()` this is called from catch blocks that handle the exception.

In `WindowManagementTrace` it is called as the exception is propagated to the caller.